flake.nix

```diff
@@ -22,10 +22,6 @@
       url = "github:berkeley-abc/abc/48498af";
       flake = false;
     };
-    bitwuzla_unstable_2021_07_01 = {
-      url = "github:bitwuzla/bitwuzla/58d720598e359b1fdfec4a469c76f1d1f24db51a";
-      flake = false;
-    };
```

Comment:
I removed this as an input, since I don't think we want to keep the unstable version around any more. That being said, I'm unclear if we should replace it with a `bitwuzla-0.3.0`–specific input of some kind.